platform/broadcom/sonic-platform-modules-cel/silverstone/modules/switchboard.c

```diff
@@ -1647,12 +1647,12 @@ static struct platform_device silverstone_dev = {
 };
 
 /**
- * Board info for QSFP/SFP+ eeprom.
+ * Board info for QSFPDD/SFP+ eeprom.
  * Note: Using OOM optoe as I2C eeprom driver.
  * https://www.opencompute.org/wiki/Networking/SpecsAndDesigns#Open_Optical_Monitoring
  */
 static struct i2c_board_info sff8436_eeprom_info[] = {
-    { I2C_BOARD_INFO("optoe1", 0x50) }, //For QSFP w/ sff8436
```

Comment:
why we replacing this not adding new an option for QSFPDD?  Do we not need "optoe1" any longer?

Also, without this, how did you get the test results for https://github.com/zhenggen-xu/sonic-buildimage/pull/89 ?

Reply:
Just use optoe3 is enough, 
And the result from https://github.com/zhenggen-xu/sonic-buildimage/pull/89 are from my local branch that already included this change.

Reply:
This optoe3 built in a kernel  https://github.com/zhenggen-xu/sonic-buildimage/commit/f08b9b53ccdaeacabdab68d404e136150c5acf09 that you included to your branch.

This optoe3 defines a larger number of eeprom page that will cover the qsfp eeprom page.
https://github.com/opencomputeproject/oom/tree/master/optoe

Reply:
I think my questions were not answered.  I understood the options added in kernel,  my question was: do we not support QSFP devices (not CMIS) any more?  Which I am fine with but just need a direct answer.

```
Use 'optoe1' to indicate this is a device with just one i2c address which
conforms to the sff-8436 spec (QSFP type devices, but not CMIS deviecs
such as QSFP-DD, OSFP, COBO, SFP-DD).

Use 'optoe2' to indicate this is a device with two i2c addresses (all
SFP type devices).

Use 'optoe3' to indicate this is a CMIS device.  CMIS devices conform to
the CMIS standard and include QSFP-DD, OSFP, COBO and SFP-DD devices.
```

Reply:
I mean 'optoe3' also supported QSFP devices